docker/Dockerfile

```
@@ -0,0 +1,27 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Build for us-docker.pkg.dev/cloud-build-fhir/openmrs/openmrs-reference-application-distro:latest image
+
+FROM openmrs/openmrs-reference-application-distro:nightly
```

Is there any specific reason that we want to use the `nightly` version? If not, I would rather we use a more stable version; please also recreate the image in GCP registry if you make this change:
```suggestion
FROM openmrs/openmrs-reference-application-distro:latest
```

---

The `nightly` version has the FHIR 1.2.2  and OCL modules and is based on Reference-app v2.14. `Latest` does not as its based on Reference-app v2.13

---

I see, let's make sure that we update this once 2.14 is released. Maybe add a TODO here documenting what you said.